src/Entity/SocialAuth.php

```diff
@@ -20,7 +20,8 @@
  *     "id" = "id",
```

Comment:
Is there a problem with this file? PHPCS is not showing any violation for this particular file. Or were you making a general comment.